### PR TITLE
remove bad practice code and add tx

### PR DESCRIPTION
**Pull request by mikedoan**
remove bad practice code and add tx.

### CODE REVIEW

**Comment by mikedoan** [comment]
Jenkins, please test this.

**Comment by mikedoan** [comment]
Jenkins, test this please.

File: node/node.go [hunk 1]
```diff
@@ -221,14 +221,20 @@ func (node *Node) Beaconchain() *core.BlockChain {
...
 	node.pendingTxMutex.Lock()
 	node.pendingTransactions = append(node.pendingTransactions, newTxs...)
 	node.pendingTxMutex.Unlock()
 	utils.GetLogInstance().Debug("Got more transactions", "num", len(newTxs), "totalPending", len(node.pendingTransactions))
 }
 
+// AddPendingTransaction adds one new transaction to the pending transaction list.
+func (node *Node) AddPendingTransaction(newTx *types.Transaction) {
+	node.addPendingTransactions(types.Transactions{newTx})
+	utils.GetLogInstance().Debug("Got ONE more transaction", "totalPending", len(node.pendingTransactions))
```
Comment (rlan35) [inline node/node.go]: lower case "one"

Comment (mikedoan) [reply node/node.go]: I'm adding in follow-up to fix comments for Richard code as EK is asking - jenkins and travis has been unstable - i want to get this in

**Comment by mikedoan** [comment]
Thanks EK. I'm making another PR to clarify on behalf of Richard.

**Pull request merged**